int32[] positions
int32[] speeds
int32[] currents
int32[] waters
int32[] temps
int32[] voltages
float64 master_voltage
float64 master_temp
float64 master_current
int32[] speedErrors
